app/views/application/_footer.html.erb

```diff
@@ -1,21 +1,26 @@
 <div class="footer valign-wrapper">
 	<div class="footer-container">
```

Comment:
@andela-tfowotade indentation looks pretty wacky, please use 2 spaces instead of four for all files including `.html.erb` ones. will also put this in the channel